target/capstone-1/index.html

```diff
@@ -0,0 +1,8 @@
+<html>
```

Comment:
Why a duplicate index.html?

Reply:
This file is auto generated by the maven appengine.

Reply:
You should not check in generated files. Only the source code that when run generates all the required files.

Reply:
This can also be included on .gitignore